Content for requests to Safe ecosystem foundation.md

```diff
@@ -0,0 +1,70 @@
+### Forms
+
+**Form on https://safe.global/**
+
+Contact us
+https://notionforms.io/forms/get-in-touch-with-the-safe-ecosystem-foundation
+
+Submit your project (for technical integrations only)
+https://github.com/safe-global/safe-ecosystem-database/issues/new?assignees=&labels=project+%3Agame_die%3A&template=suggest_project.yaml&title=Suggest+a+project
+
+**Forms on https://www.gnosis.builders/contact-us**
+
+Collaborate with us
+https://forms.monday.com/forms/8636cfde490a3ce083f083e5376487a9?r=use1
+
+Connect with the team 
+https://forms.monday.com/forms/37fe262572726e8f752319bea04448bd?r=use1
+
+---
+
+### 1. Safe to provide bounties for DevPool
+
+**COMPANY/DAO/PROJECT NAME**
+
+DevPool by Ubiquity DAO
+
+**COMPANY/PROJECT WEBSITE** 
+
+https://github.com/ubiquity/devpool-directory
+
+**PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES, WHAT YOU WANT TO DO WITH THE SAFE, AND WHERE THE CORE CONTRIBUTORS CAN HELP (150 CHARACTERS MINIMUM)** 
+
+DevPool is a Web3 freelance marketplace on top of GitHub. 
+It scales development throughput with talents available on GitHub and automates routine freelance management activities.
+Ubiquity DAO uses Gnosis Chain for payouts to DAO members and freelancers.
+
+DevPool is in beta testing with the open issues listed here https://github.com/ubiquity/devpool-directory
+Currently, the cost for DevPool services is only payout for freelancers without any fees for the DevPool.
+We offer that DevPool will solve several issues currently open in your GitHub projects https://github.com/safe-global https://github.com/gnosischain
+
+Looking forward to your feedback.
+
+---
+
+### 2. Ubiquity to create a Dapp to the Safe marketplace
+
+**COMPANY/DAO/PROJECT NAME**
+
+Ubiquity DAO
+
+**COMPANY/PROJECT WEBSITE** 
+
+https://ubq.fi/
+
+**PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES, WHAT YOU WANT TO DO WITH THE SAFE, AND WHERE THE CORE CONTRIBUTORS CAN HELP (150 CHARACTERS MINIMUM)** 
+
+Ubiquity DAO is building an ecosystem bridging the gap between Web3 and traditional economies.
+There is a stablecoin, a freelance marketplace, and a payment card among the Ubiquity products.
+Ubiquity DAO uses Gnosis Chain for payouts to DAO members and freelancers.  
+
+We offer Gnosis and Safe Ecosystem to partner in the creation of the Safest crypto card.
+
+Users' funds will be protected with the complete set of security features:
+- Using the Safe crypto wallet
+- Shielding the card information with temporary virtual cards (as on https://privacy.com/)
+- Ability to configure card transaction multisig with on-chain and common 2FA methods (a customizable analog of 3DS)
+- Abilitity to configure white- and black-lists, as well as spending limits for transactions (by MCC, countries, specific destinations)
+- Ability to protect funds by temporary card freezing in case or customizable fraud triggers
```

Comment:
You're speaking as if we already have the card available. Perhaps its more accurate to state that it'll be available soon™️

Reply:
Thank you, updated to highlight that the card is planned